### ISSUE TITLE
accuracy not improving

### ISSUE THREAD

**Issue by vinayakumarr** [albertbup/deep-belief-network#6]
I have data that contains the probability values. It is a 4 class classification problem. The accuracy is not improving with your classification example. The data format is given below
![untitled](https://cloud.githubusercontent.com/assets/18217467/23748328/2e139034-04e9-11e7-857b-917dadfafd72.png)

**Comment by albertbup** [comment]
Hey,
If I've understood you well, you've run classification_demo.py script with your own data and what has happened is that the accuracy was not good, is it correct? That's pretty normal, every dataset has their own optimal hyperparameters (number of hidden layers, learning rate, dropout etc), so you must find the optimal ones for your data.

Best,
Albert.

**Comment by lhr5687** [comment]
Hey
   Mr albertbup，thank you for your reply。
_________
Hongri Liu 
Harbin Institute of Technology（Wei Hai）


 
From: albertbup
Date: 2017-03-10 03:31
To: albertbup/deep-belief-network
CC: Subscribed
Subject: Re: [albertbup/deep-belief-network] accuracy not improving (#6)
Hey,
If I've understood you well, you've run classification_demo.py script with your own data and what has happened is that the accuracy was not good, is it correct? That's pretty normal, every dataset has their own optimal hyperparameters (number of hidden layers, learning rate, dropout etc), so you must find the optimal ones for your data.
Best,
Albert.
